core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/rules/BundleInlineColumnExpressions.scala

```diff
@@ -0,0 +1,47 @@
+package com.databricks.labs.remorph.parsers.snowflake.rules
+
+import com.databricks.labs.remorph.intermediate._
+
+class BundleInlineColumnExpressions extends Rule[LogicalPlan] {
+
+  private val inlineBundleTable = "INLINE_BUNDLE"
```

Comment:
there might be many different names in CTE. Add a new tree node to wrap this name and change the names before they get to codegen.

Reply:
and apparently, we don't need this anymore :)